client-gateway/bot/drive_ops_bot.py

```
@@ -0,0 +1,225 @@
+load_dotenv()
+BOT_TOKEN = os.getenv('BOT_TOKEN')
+
+# Fail fast with a clear message if BOT_TOKEN is not set
+if not BOT_TOKEN:
+    logging.error("BOT_TOKEN is not set in the environment or .env file. Please set BOT_TOKEN and restart the bot.")
+    sys.exit("ERROR: BOT_TOKEN is not configured. Please set BOT_TOKEN in the environment or in the .env file.")
+
+# Initialize bot only after validation
+bot = telebot.TeleBot(BOT_TOKEN)
+
+# Тимчасове сховище для замовлень (у пам'яті)
```

Ruff (0.14.10)

Comment contains ambiguous `у` (CYRILLIC SMALL LETTER U). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF003)